src/index.html

```diff
@@ -3,11 +3,444 @@
+                  class="nav__link"
+                >
+                  Blog
+                </a>
+              </li>
+              <li class="nav__item nav__item--4">
+                <a
+                  href="tel:6295550129"
+                  class="nav__link nav__link--active"
+                >
+                  <img
+                    src="images/icon/icon-phone.png"
+                    alt="phone"
+                    class="icon icon-phone">
+                  Contact us
+                </a>
+              </li>
+
+
+              <li class="nav__item">
+                <a
+                  href="#menu"
+                  class="nav__link nav__link--menu">
+                  <img
+                    src="images/icon/icon-menu.png"
+                    alt="menu"
+                    class="icon icon-menu">
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+    </header>
+
+    <aside class="menu__page" id="menu">
+      <div class="top--bar top--bar-menu">
+        <a href="#" class="top--bar__logo-link">
+          <img
+            src="images/top-bar__logo.png"
+            alt="Creative__Bakery"
+            class="top--bar__logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link nav__link--close">
+                <img
+                  src="images/icon/icon-close.png"
+                  alt="close"
+                  class="icon icon-close">
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+      <div class="menu__category-wrapper">
+        <a href="#product" class="menu__category">Products</a>
+        <a href="#about-us" class="menu__category">About Us</a>
+        <a href="https://www.facebook.com/ViCake.Creative.Bakery/?locale=ru_RU" class="menu__category">Blog</a>
+        <a href="tel:6295550129" class="menu__info--link">
+          <img
+           src="images/icon/icon-phone.png"
+           class="icon icon-phone"
+           alt="phone"
+          >
+          <p class="menu_info">Contact us</p>
+        </a>
+      </div>
+    </aside>
+
+
+    <main class="main">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__picture-container">
+            <div class="header__picture">
+              <div class="header__info-wrapper">
+                <p class="info info--header">We bake your fantasies — croissants, cakes, cookies, and cupcakes.</p>
+              </div>
+              <div class="header__text-wrapper">
+                <h1 class="header__text">Creative bakery</h1>
+
+                <img
+                src="images/cookie.png"
+                alt="coockie"
+                class="header__coockie"
+                >
```

Comment:
Need to check and fix it everywhere
```suggestion
                <img
                  src="images/cookie.png"
                  alt="coockie"
                  class="header__coockie"
                >
```